pkg/build/builder/sti.go

```diff
@@ -128,7 +128,7 @@ func makeVolumeSpec(src localObjectBuildSource, mountPath string) s2iapi.VolumeS
 
 // Build executes S2I build based on configured builder, S2I builder factory
 // and S2I config validator
-func (s *S2IBuilder) Build() error {
+func (s *S2IBuilder) Build(getContainerNetworkConfigFunc GetContainerNetworkConfigFn) error {
```

Comment:
could you split it into two functions?

an internal one `buildInternal(getContainerNetworkConfigFunc GetContainerNetworkConfigFn` that would be used by the tests

and an external one `Build() { buildInternal(getContainerNetworkConfig()}` that would platform provided function.

Reply:
agreed. i made the change. please take a look.